v2/api/cognitiveservices/v1api20241001/structure.txt

```diff
@@ -0,0 +1,431 @@
+    │   ├── ProvisioningState: *Enum (7 values)
+    │   │   ├── "Accepted"
+    │   │   ├── "Creating"
+    │   │   ├── "Deleting"
+    │   │   ├── "Failed"
+    │   │   ├── "Moving"
+    │   │   ├── "ResolvingDNS"
+    │   │   └── "Succeeded"
+    │   ├── PublicNetworkAccess: *Enum (2 values)
+    │   │   ├── "Disabled"
+    │   │   └── "Enabled"
+    │   ├── QuotaLimit: *Object (3 properties)
+    │   │   ├── Count: *float64
+    │   │   ├── RenewalPeriod: *float64
+    │   │   └── Rules: Object (6 properties)[]
+    │   │       ├── Count: *float64
+    │   │       ├── DynamicThrottlingEnabled: *bool
+    │   │       ├── Key: *string
+    │   │       ├── MatchPatterns: Object (2 properties)[]
+    │   │       │   ├── Method: *string
+    │   │       │   └── Path: *string
+    │   │       ├── MinCount: *float64
+    │   │       └── RenewalPeriod: *float64
+    │   ├── RaiMonitorConfig: *Object (2 properties)
+    │   │   ├── AdxStorageResourceId: *string
+    │   │   └── IdentityClientId: *string
+    │   ├── Restore: *bool
+    │   ├── RestrictOutboundNetworkAccess: *bool
+    │   ├── ScheduledPurgeDate: *string
+    │   ├── SkuChangeInfo: *Object (3 properties)
+    │   │   ├── CountOfDowngrades: *float64
+    │   │   ├── CountOfUpgradesAfterDowngrades: *float64
+    │   │   └── LastChangeDate: *string
+    │   └── UserOwnedStorage: Object (2 properties)[]
+    │       ├── IdentityClientId: *string
+    │       └── ResourceId: *string
+    ├── Sku: *Object (5 properties)
+    │   ├── Capacity: *int
+    │   ├── Family: *string
+    │   ├── Name: *string
+    │   ├── Size: *string
+    │   └── Tier: *Enum (5 values)
+    │       ├── "Basic"
+    │       ├── "Enterprise"
+    │       ├── "Free"
+    │       ├── "Premium"
+    │       └── "Standard"
+    ├── SystemData: *Object (6 properties)
+    │   ├── CreatedAt: *string
+    │   ├── CreatedBy: *string
+    │   ├── CreatedByType: *Enum (4 values)
+    │   │   ├── "Application"
+    │   │   ├── "Key"
+    │   │   ├── "ManagedIdentity"
+    │   │   └── "User"
+    │   ├── LastModifiedAt: *string
+    │   ├── LastModifiedBy: *string
+    │   └── LastModifiedByType: *Enum (4 values)
+    │       ├── "Application"
+    │       ├── "Key"
+    │       ├── "ManagedIdentity"
+    │       └── "User"
+    ├── Tags: map[string]string
+    └── Type: *string
+
+Deployment: Resource
+├── Owner: Account
+├── Spec: Object (6 properties)
+│   ├── AzureName: string
+│   ├── OperatorSpec: *Object (2 properties)
+│   │   ├── ConfigMapExpressions: *core.DestinationExpression[]
+│   │   └── SecretExpressions: *core.DestinationExpression[]
+│   ├── Owner: *genruntime.KnownResourceReference
+│   ├── Properties: *Object (7 properties)
+│   │   ├── CapacitySettings: *Object (2 properties)
+│   │   │   ├── DesignatedCapacity: Validated<*int> (1 rule)
+│   │   │   │   └── Rule 0: Minimum: 0
+│   │   │   └── Priority: Validated<*int> (1 rule)
+│   │   │       └── Rule 0: Minimum: 0
+│   │   ├── CurrentCapacity: *int
```

Comment:
Is this field actually writable? Its name makes me think it's readonly? I'm wondering if we should prune it from the spec?